src/behavior/src/A32NX_Interior_Autopilot.xml

```diff
@@ -115,7 +119,7 @@
             <TOOLTIPID>%((L:#PLANE_NAME#_AUTOPILOT_APPR_MODE, bool))%{if}TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_APR_ON%{else}TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_APR_OFF%{end}</TOOLTIPID>
         </DefaultTemplateParameters>
 
-        <UseTemplate Name="ASOBO_AUTOPILOT_Push_Template">
```

Comment:
In some files `FBW` is used as a template name prefix. In others `A32NX` is used. Here we are in a `A32NX` file, which has templates specific to `A32NX`.

What is the standard and why?

Reply:
@Benjozork says FBW for new ones https://github.com/flybywiresim/a32nx/pull/4381#discussion_r615343377

Reply:
@Benjozork While the idea of template reuse is a noble one, there are many locations where reuse isn't assumed. As an example, `A32NX_Interior_Generics.xml` (which is prefixed with A32NX and therefore _seems_ A32NX specific) contains references to `(L:A32NX_BRAKES_HOT, Bool)` and `(L:A32NX_BRAKE_FAN, Bool)` in a `FBW` prefixed template (`FBW_Push_BrkFan_Toggle`).

For now, I'll rename any template I added to use the `FBW` prefix. If I added a usage of an `A32NX` LVar to a template, and there wasn't already an `A32NX` LVar in it, then I'll try to move those `A32NX` usages out of the template into the `A32NX` "level".

`A32NX` and generic templates are mixed. We need to provide more guidance and move templates outside of `flybywire-aircraft-a320-neo`. Otherwise any other developer will just "fight" with the same issue.

Reply:
Yes, I've wondered about the brake fan template in the generics file as well, that should be removed entirely and the brake fan button replaced with the generic template. But that is a matter for another PR

Reply:
Some templates I made use a `#PLANE_NAME#` "global parameter" defined in a root component. Could be explored in the future.